test/tests/functional/pbs_reservations.py

```diff
@@ -120,6 +120,35 @@ def test_degraded_standing_reservations(self):
         self.server.expect(RESV, a, id=rid, interval=1, offset=offset,
                            attrop=PTL_AND)
 
+    def test_duration_end_resv(self):
+        """
+        To test if reservations can be made by using
+        Duration and EndTime, making the server calculate
+        the StartTime.
```

Comment:
Duration, EndTime, and StartTime are not variable names.  I wouldn't camelcase them.